### ISSUE TITLE
icvCreateTrackbar error

### ISSUE THREAD

**Issue by mertmzk** [AlejandroSilvestri/os1#5]
Hi Alejandro,

While I am triying to run your code on Eclipse but I am getting this error:

VIDEOIO ERROR: V4L2: Unable to get property <unknown property string>(7) - Invalid argument
OpenCV Error: One of arguments' values is out of range (Bad trackbar maximal value) in icvCreateTrackbar,

Do you know what might be reason for that?

Thanks

**Comment by AlejandroSilvestri** [comment]
@mertmzk ,

One trackbar is the video timeline (if you are getting your video from a file, not live).  I believe that is the property v4ls is unable to get.

I suppose you are not using a video file, but a webcam.  If so, I'll test it myself to see what is happening.

It helps a lot if you can run it in debug mode, and tell me what line in src files is initiating that error.

**Comment by mertmzk** [comment]
Hi Alejandro,

Thanks for your response. I have found the problem that is related to my webcam specs as it does not support feature of "CV_CAP_PROP_FRAME_COUNT". I have manually set them and it is working now.

But I have got other problems I would be grateful if you answer them as well. First, when I click reset button the program is freezing which is not happening with original ORB_SLAM2. It also happens when I try to load a previously saved map. Saving a mapping seems to work but it fails (freezes) when loading a saved map. Have you sorted that out?

Regards,
